prompt_toolkit/shortcuts/progress_bar/base.py

```diff
@@ -361,18 +364,49 @@ def item_completed(self) -> None:
 
     @property
     def done(self) -> bool:
+        """Whether a counter has been completed.
+
+        Done counter have been stopped (see stop) and removed depending on
+        remove_when_done value.
+
+        Contrast this with stop. A stopped counter may be terminated before
+        100% completion. A done counter has reached its 100% completion.
+        """
         return self._done
 
     @done.setter
     def done(self, value: bool) -> None:
         self._done = value
-
-        # If done then store the stop_time, otherwise clear.
-        self.stop_time = datetime.datetime.now() if value else None
+        self.stop = value
 
         if value and self.remove_when_done:
             self.progress_bar.counters.remove(self)
 
+    @property
+    def stop(self) -> bool:
```

Comment:
Should we call it ``stopped``, because it's more like a state, so a noun instead of a verb, like ``done``.

Reply:
No problem